doc/user/content/demos/business-intelligence.md

```diff
@@ -7,7 +7,7 @@ menu:
         weight: 4
 ---
 
-**tl;dr** Materialize can enable real-time monitoring within business
+**tl;dr** Materialize can enable real-time data dashboards within business
```

Comment:
I believe monitoring was the correct term here because monitoring can happen while no one is watching, whereas data dashboard feel like a primarily GUI-driven tool.